client/app/collections/duplicates.coffee

```diff
@@ -7,11 +7,15 @@ module.exports = class Duplicates extends Backbone.Collection
 
     model: require 'views/models/merge'
 
-    url: null # Local collection.
+    # Local collection
+    url: null
 
     # Initialize the collection with duplicates fund in the sepcified contacts
     # collection.
-    findDuplicates: (collection) ->
+    initialize: ([], options) ->
```

Comment:
I don't understand this line. Can you explain me what the `[]` means here? It's the default for something?